diff --git a/node_modules/style-dictionary/index.js b/node_modules/style-dictionary/index.js
index 14ec78a..a376877 100644
--- a/node_modules/style-dictionary/index.js
+++ b/node_modules/style-dictionary/index.js
@@ -15,6 +15,7 @@ var GroupMessages = require('./lib/utils/groupMessages');
 var TEMPLATE_DEPRECATION_WARNINGS = GroupMessages.GROUP.TemplateDeprecationWarnings;
 var REGISTER_TEMPLATE_DEPRECATION_WARNINGS = GroupMessages.GROUP.RegisterTemplateDeprecationWarnings;
 var SASS_MAP_FORMAT_DEPRECATION_WARNINGS = GroupMessages.GROUP.SassMapFormatDeprecationWarnings;
+var process = require('process');
 
 /**
  * Style Dictionary module
diff --git a/node_modules/style-dictionary/lib/extend.js b/node_modules/style-dictionary/lib/extend.js
index 0a8ed7a..a46f459 100644
--- a/node_modules/style-dictionary/lib/extend.js
+++ b/node_modules/style-dictionary/lib/extend.js
@@ -11,8 +11,10 @@
  * and limitations under the License.
  */
 
-require('json5/lib/register');
-require.extensions[".jsonc"] = require("./utils/jsonc").register;
+if (typeof require !== 'undefined' && require.extensions) {
+  require('json5/lib/register');
+  require.extensions[".jsonc"] = require("./utils/jsonc").register;
+}
 
 var combineJSON = require('./utils/combineJSON'),
     deepExtend = require('./utils/deepExtend'),
diff --git a/node_modules/style-dictionary/lib/utils/combineJSON.js b/node_modules/style-dictionary/lib/utils/combineJSON.js
index 1b7329e..a319002 100644
--- a/node_modules/style-dictionary/lib/utils/combineJSON.js
+++ b/node_modules/style-dictionary/lib/utils/combineJSON.js
@@ -11,8 +11,10 @@
  * and limitations under the License.
  */
 
-require('json5/lib/register');
-require.extensions[".jsonc"] = require("./jsonc").register;
+if (typeof require !== 'undefined' && require.extensions) {
+  require('json5/lib/register');
+  require.extensions[".jsonc"] = require("./jsonc").register;
+}
 
 var glob = require('glob'),
   deepExtend = require('./deepExtend'),
